### PR TITLE
3.2 aa feat refonte login

### PR DESCRIPTION
**Pull request by anaKnell**

### CODE REVIEW

File: skillful_network_client/src/app/login/login.component.ts [hunk 1]
```diff
@@ -132,4 +123,15 @@ export class LoginComponent implements OnInit {
...
+
+  snackBarNewCodeSend() {
+    this._snackBar.open("Vous avez reçu un nouveau code", "X", {
+      duration: 1500,
+      verticalPosition: 'bottom'
+    });
+  }
+
+  reloadPage() {
+    setTimeout(function () { window.location.reload(); }, 3000);
```
Comment (danglotb) [inline skillful_network_client/src/app/login/login.component.ts]: Is it necessary to reload the page? Could we reinit it?

**Review by danglotb — CHANGES_REQUESTED** [review]

File: skillful_network_client/src/app/login/login.component.ts (hunk 1 above)
Comment (anaKnell) [reply skillful_network_client/src/app/login/login.component.ts]: Maybe like this : I replaced method onload by ngOnInit() to reinit and I removed the condition of [disabled]= "submitted" from the button on this form.

commit =>[replace reload to ngOnInit()](https://github.com/danglotb/skillful_network/pull/45/commits/b4909e4ac407def65bdd1ca079baa7c83c087f90)

**Review by danglotb — APPROVED** [review]

**Review by danglotb — APPROVED** [review]

**Review by danglotb — APPROVED** [review]

**Comment by danglotb** [comment]
Tant pis pour le coverage, on le remontrera quand on pourra.

**Comment by danglotb** [comment]
On peut merge ?

**Pull request merged**